homeassistant/components/spotify/media_player.py

```diff
@@ -105,18 +105,18 @@
 }
 
 CONTENT_TYPE_MEDIA_CLASS = {
-    "current_user_playlists": MEDIA_CLASS_PLAYLIST,
-    "current_user_followed_artists": MEDIA_CLASS_ARTIST,
-    "current_user_saved_albums": MEDIA_CLASS_ALBUM,
-    "current_user_saved_tracks": MEDIA_CLASS_TRACK,
-    "current_user_saved_shows": MEDIA_CLASS_PODCAST,
-    "current_user_recently_played": MEDIA_CLASS_TRACK,
-    "current_user_top_artists": MEDIA_CLASS_ARTIST,
-    "current_user_top_tracks": MEDIA_CLASS_TRACK,
-    "featured_playlists": MEDIA_CLASS_PLAYLIST,
-    "categories": MEDIA_CLASS_GENRE,
-    "category_playlists": MEDIA_CLASS_PLAYLIST,
-    "new_releases": MEDIA_CLASS_ALBUM,
+    "current_user_playlists": MEDIA_CLASS_DIRECTORY,
```

Comment:
@bramkragten is this according to what you'd expect? We keep the directory with multiple playlists classified as a non specific directory, while its children will be classified as media class playlist.

Reply:
If we can see the type of it' content by the children's class, then yes this would be most consistent.

Reply:
Good!